docs/changes/0.20.0.rst

```diff
@@ -1,43 +1,76 @@
-Changelog for QCoDeS 0.20.0 (unreleased)
-========================================
+Changelog for QCoDeS 0.20.0
+===========================
 
 The November 2020 release of QCoDeS.
 
-
+-----------------
 Breaking Changes:
-_________________
-
-When performing a measurement with ``MultiParameter``
-the parameters are now registered under their full_name including
-any instrument and channels that the parameter is bound to.
-This makes the ``MultiParameter`` consistent with all other parameters (#2317)
+-----------------
 
+- When performing a measurement with ``MultiParameter`` the parameters are now registered under their ``full_name`` including any instrument and channels that the parameter is bound to. This makes the ``MultiParameter`` consistent with all other parameters (#2317)
+- The Keysight ``SD_common`` module is now deprecated in favour of the one in ``qcodes_contrib_drivers`` (#2395)
 
+----
 New:
-____
-
+----
+- New support email introduced (#2366)
+- The ``Measurement`` now captures exception info in the ``DataSet`` s metadata if an exception occured during measurement (#2327)
 
+---------
 Improved:
-_________
-
-
+---------
+- The documentation for ``ParameterWithSetpoints`` now shows how to make sweeps (#2351)
+- There is now an explicit API for marking the ``cache`` of a ``Parameter`` as invalid (#2012)
+- Bugfix: calling ``mark_complete()`` on a completed ``DataSet`` no longer throws an exception (#2376)
+- The documentation no longer references the legacy module (#2408)
 
 
+------------
 New drivers:
-____________
-
-
+------------
+- Keysight E4980A Precision LCR Meter (#2253)
 
 
+-----------------
 Improved drivers:
-_________________
-
-
-
+-----------------
+- The Alazar drivers received various minor fixes (#2301)
+- The Rohde & Schwarz ZNB vector network analyzer now supports CW readout mode (#2007)
+- The Keithley 7510 now has a settable buffer name (#2370)
+- Bugfix: Ithaco ``CurrentParameter`` (#2416)
+- Keysight B1500 no longer has master/slave terminology in its API (#2423)
+- Agilent E8527D now has a default terminator (#2434)
+
+---------------
 Under the hood:
-_______________
-
-
-
-Dependencies:
-_____________
+---------------
+
+- The ``DataSet`` shape is used when loading data (#2264)
```

Comment:
i'm tempted to put in in the "new" and mention that DataSet.get_parameter_data and DataSet.cache.data are the "affected" methods

Reply:
Agreeing with @astafan8 I would perhaps also put a link to the relevant part of the "performing measurements with ...` notebook